src/core/Microsoft.PowerToys.Settings.UI/Views/KeyboardManagerPage.xaml

```diff
@@ -0,0 +1,244 @@
+                           FontWeight="Bold"/>
+                <TextBlock Text="Config-1"
+                           Margin="{StaticResource XSmallLeftMargin}"
+                           FontWeight="ExtraLight" />
+            </StackPanel>
+
+            <StackPanel Margin="{StaticResource SmallLeftMargin}"
+                        Orientation="Horizontal">
+                <TextBlock Text="From:"
+                           Width="200"/>
+                <TextBlock Text="To:"
+                           Width="200"
+                           FontWeight="ExtraLight" />
+            </StackPanel>
+
+            <ListView x:Name="RemapKeysList"
+                      ItemsSource="{StaticResource dummyData}"
+                      BorderBrush="LightGray"
+                      BorderThickness="1" 
+                      Width="400"
+                      MaxHeight="150"
+                      Margin="{StaticResource SmallLeftTopRightBottomMargin}"
+                      HorizontalAlignment="Left">
+                <ListView.ItemTemplate>
+                    <DataTemplate>
+                        <Grid>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="200"/>
+                                <ColumnDefinition Width="200"/>
+                            </Grid.ColumnDefinitions>
+                            <Border Background="{ThemeResource SystemBaseLowColor}"
+                                    Grid.Column="0"
+                                    Width="70"
+                                    Height="35"
+                                    HorizontalAlignment="Left">
+                                <TextBlock  Grid.Column="0"
+                                            Style="{StaticResource BodyTextBlockStyle}"
+                                            VerticalAlignment="Center"
+                                            TextAlignment="Center"
+                                            Text="{Binding From}">
+                                </TextBlock>
+                            </Border>
+                            <Border Background="{ThemeResource SystemBaseLowColor}"
+                                    Grid.Column="1"
+                                    Width="70"
+                                    Height="35"
+                                    HorizontalAlignment="Left">
+                                <TextBlock Grid.Column="1" 
+                                           Style="{StaticResource BodyTextBlockStyle}"
+                                           VerticalAlignment="Center"
+                                           TextAlignment="Center"
+                                           Text="{Binding To}">
+                                </TextBlock>
+                            </Border>
+                        </Grid>
+                    </DataTemplate>
+                </ListView.ItemTemplate>
+            </ListView>
```

Comment:
This looks really neat in the UI, and also solves the issue of scrolling when the number of remaps overflows in the page. We should do the same for the Edit Keyboard and Edit Shortcuts window as well.

Reply:
Agree we should similar layout for the `Edit Keyboard` and `Edit Shortcut` windows. We can also leverage other styles like margins and font size from the main settings to be consistent